matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/model/Membership.kt

```diff
@@ -17,10 +17,12 @@
 package im.vector.matrix.android.api.session.room.model
 
 import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 
 /**
  * Represents the membership of a user on a room
  */
+@JsonClass(generateAdapter = false)
```

Comment:
I assume it's necessary for PG, but why `false`? This is the first time we use `false` :)

Reply:
It's the rule for proguard recommanded by Moshi devs. It's to make it clear it shouldn't be removed but we don't need adapter as it use the EnumJsonAdapter from Moshi.